src/SmartComponents/BaselinesPage/EditBaseline/EditBaselineToolbar/EditBaselineToolbar.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Button, Toolbar, ToolbarGroup, ToolbarItem } from '@patternfly/react-core';
+
+import { factModalActions } from '../FactModal/redux';
+
+class EditBaselineToolbar extends Component {
+    constructor(props) {
+        super(props);
+
+        this.handleAddFact = this.handleAddFact.bind(this);
+    }
+
+    handleAddFact() {
+        const { setFactData, toggleFactModal } = this.props;
+
+        setFactData({ factName: '', valueName: '', fact: []});
```

Comment:
Nitpick -- unbalanced spacing after `{` and before `}`

Reply:
If I try to add a space there, eslint tells me I don't need a space there.
```
/home/micjohns/Desktop/InsightsApp/drift-frontend/src/SmartComponents/BaselinesPage/EditBaseline/EditBaselineToolbar/EditBaselineToolbar.js
  18:61  error  There should be no space before '}'  object-curly-spacing
```

Reply:
What about getting rid of the space after `{` ?